scripts/scil_visualize_fodf.py

```diff
@@ -0,0 +1,325 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Visualize 2-dimensional fODF slice loaded from disk.
+
+Given a SH coefficients image, this script displays a slice in the
+orientation specified by the user. The user can also add a background
+on top of which the fODF are to be displayed.
+"""
+
+import argparse
+
+import nibabel as nib
+import numpy as np
+
+from dipy.data import get_sphere
+from dipy.core.sphere import Sphere
+from dipy.reconst.shm import sh_to_sf
+from fury import window, actor
+
+from scilpy.io.utils import (add_sh_basis_args, add_overwrite_arg,
+                             assert_inputs_exist, assert_outputs_exist)
+
+# Default background offset when none is specified
+DEFAULT_OFFSET = 0.5
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    # Positional arguments
+    p.add_argument('input_fodf', type=str, help='Input SH image file')
+
+    p.add_argument('slice_index', type=int,
```

Comment:
I would put this argument optional, with the default (None) being the middle of the volume. Depends on what the other guys think of it.

Reply:
No problem, I could add the change to my next commit if no one objects.